recipes/elixir/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "1.9.4" %}
+
+package:
+  name: elixir
+  version: {{ version }}
+
+source:
+  url: https://github.com/elixir-lang/elixir/archive/v{{ version }}.tar.gz
+  sha256: f3465d8a8e386f3e74831bf9594ee39e6dfde6aa430fe9260844cfe46aa10139
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: make install
```

Comment:
Does this need `--prefix=$PREFIX`?

Reply:
Do i have to worry about the other FLAGs i wonder? I'm such a `make` charlatan...

Reply:
This is usually enough, unless you are putting stuff in weird places ;)